api/src/pcapi/alembic/versions/20220922T123207_6858182908fe_delete_enable_phone_in_stepper_flag.py

```diff
@@ -0,0 +1,24 @@
+"""delete_enable_phone_in_stepper_flag
+"""
+from alembic import op
+
+
+# pre/post deployment: post
+# revision identifiers, used by Alembic.
+revision = "6858182908fe"
+down_revision = "898cc1f49e37"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.execute("DELETE FROM feature where name = 'ENABLE_PHONE_VALIDATION_IN_STEPPER'")
+
+
+def downgrade() -> None:
+    op.execute(
+        """
+        INSERT INTO feature (name, description, "isActive")
+        VALUES ('ENABLE_PHONE_VALIDATION_IN_STEPPER', 'Déplace la validation du numéro de téléphone dans le flux du parcours d inscription', false)
```

Comment:
Hmm, actuellement le feature flag est **activé**. Si on rollback cette migration, on devrait remettre le flag à sa valeur précédente, non ?

```suggestion
        VALUES ('ENABLE_PHONE_VALIDATION_IN_STEPPER', 'Déplace la validation du numéro de téléphone dans le flux du parcours d inscription', true)
```

Reply:
Bonne remarque !